docs/debugging.md

```diff
@@ -0,0 +1,305 @@
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/asb-installer
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/deploy-broker
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-deployer
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-docker-registry
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-haproxy-router
+[2018-01-22T21:22:13.006Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-pod
+[2018-01-22T21:22:13.007Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-sti-builder
+[2018-01-22T21:22:13.007Z] [DEBUG] - Trying to load ansibleplaybookbundle/origin-recycler
+[2018-01-22T21:22:13.007Z] [DEBUG] - Filter applied against registry: dh
+[2018-01-22T21:22:13.007Z] [DEBUG] - APBs passing white/blacklist filter:
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/pyzip-demo-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/pyzip-demo-db-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/nginx-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/manageiq-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/jenkins-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/hello-world-db-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/rds-postgres-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/etherpad-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/mediawiki-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/mysql-apb
+[2018-01-22T21:22:13.007Z] [DEBUG] - -> ansibleplaybookbundle/mariadb-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/s2i-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/hello-world-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/rocketchat-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/hastebin-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/wordpress-ha-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/postgresql-apb
+[2018-01-22T21:22:13.008Z] [DEBUG] - -> ansibleplaybookbundle/thelounge-apb
+```
+
+If you do NOT see any APBs as shown in the logs above, verify that the Docker Org used is correct, and that the permissions for the org are properly set (i.e. public).
+
+## Verify the ASB connection to the Service Catalog
+
+The ASB may get a list of APBs from the Docker Org, but it still needs to connect to the service catalog to use them in the cluster.
+
+Verify that ASB's service catalog connection was successful by making sure that no connection errors occurred (e.g. 'TLS handshake errors').  If the ASB's service catalog connection was successful, the logs may look something like this.
+
+```bash
```

Comment:
You can also verify the connection with this command:
```curl -H "authorization: bearer $(oc whoami -t)"  https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/catalog --insecure```

Reply:
thanks, I will add this

Reply:
Instead of having the log and the curl command, you can replace reading the log with the curl command. Sorry I wasn't clear by what I meant by my comment.

Reply:
OK.. I will update